config/locales/strings/en.posts.yml

```diff
@@ -49,7 +49,7 @@ en:
     redact_label: >
       Redact
     redact_explanation: >
-      Redact original content by hiding the previous versions from history?
+      Redact original content by hiding the previous versions from history? Use sparingly, for example to hide passwords or personal information.
```

Comment:
I've confirmed that the new wording shows up correctly in the user interface, so my only question is whether to fine tune the wording. It's already a big improvement, but I wonder if some people would read "Use sparingly ... " as "Use your own judgement, the rest of this sentence is just rough guidance".

We could be more imperative, such as "Use only for passwords and personally identifiable information". The problem with this is that we've moved from a list of examples to an exhaustive list, so we'd have to add in any other reasons that might ever apply.

Reply:
Good point.  I've changed that sentence to: "Use only for private information such as passwords or personally identifiable information."  That's a little more verbose but keeps the "example" framing.